static/schemas/2.0/Lookups.2.0.json

```diff
@@ -0,0 +1,25 @@
+{
+    "title": "Lookup.2.0",
+    "definitions": {
+        "Lookup": {
+            "description": "A lookup record",
+            "type": "object",
+            "properties": {
+                "node": {
+                    "type": "string"
+                },
+                "ipAddress": {
+                    "type": "string",
+                    "format": "ipv4"
```

Comment:
The ipv4 format validator needs to be added to tv4.  It can be added to tv4 by calling addFormat: "tv4.addFormat(format, validationFunction)".